prisma/entrypoint.sh

```diff
@@ -9,7 +9,8 @@ echo Start applying migrations...
 npx ts-node manual_migrations.ts
 
 # apply migration
-npx prisma migrate up -c --auto-approve --experimental
+npx prisma db push
+npx prisma migrate dev
```

Comment:
seems like `migrate dev` is for development use only 
could we use `seed` instead? https://www.prisma.io/docs/reference/api-reference/command-reference#db-seed

Reply:
Changed